app/src/main/java/com/xmartlabs/template/model/Balance.kt

```diff
@@ -1,7 +1,11 @@
 package com.xmartlabs.template.model
 
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+
+@Entity
 data class Balance(
-    var user: User? = null,
-    var favour: Int = 0,
-    var against: Int = 0
+    @PrimaryKey val balanceId: Int? = 0,
```

Comment:
i think that it would be better to put null as a default instead of 0 for all those ints